mesmerize_napari/cnmfe_gui.py

```diff
@@ -62,6 +62,7 @@ def _add_item_cnmfe(self, *args, group_params: bool = True):
             "method_deconvolution": method_deconvolution,
             "update_background_components": True,
             "del_duplicates": True,
+            "fr": self.ui.doubleSpinBoxFrameRate.value(),
```

Comment:
similar to CNMF, put a check at the top of the `get_params()` to raise if `<= 0`

Reply:
good idea